x-pack/plugins/security_solution/cypress/e2e/timelines/bulk_add_to_timeline.cy.ts

```diff
@@ -22,8 +23,9 @@ import { openEvents, openSessions } from '../../tasks/hosts/main';
 import { login, visit } from '../../tasks/login';
 import { ALERTS_URL, HOSTS_URL } from '../../urls/navigation';
 
-const assertFirstPageEventsAddToTimeline = () => {
-  selectFirstPageEvents();
+const assertFirstPageEventsAddToTimeline = (type: 'alerts' | 'events') => {
```

Comment:
We should not have conditionals inside the testing code.

Reply:
Moreover we should avoid having assertions methods in favor of having the assertions directly on the test.

Reply:
Since this utility function was already there, I am thinking that we should tackle that issue saperately. Additionally, removing this utility function will lead to lot of code repetition. 

Do you think it is a big issue if it is just a local function?